i18n-core/src/main/resources/javadoc/overview.html

```diff
@@ -1,134 +0,0 @@
-<html>
```

Comment:
This should be migrated to `package-info.java`. IMO this is CDDL content and not CCPL as this is not standard site content but JavaDoc content which is part of the codebase.

Reply:
`package-info.java` files are already present in every module and contain relevant description (e.g. [i18n-core/package-info.java](https://github.com/WrenSecurity/wrensec-i18n-framework/blob/master/i18n-core/src/main/java/org/forgerock/i18n/package-info.java)). 
Generation of sites have been fixed and the result looks fine.